src/modules/date/index.ts

```diff
@@ -190,13 +194,23 @@ export class SimpleDateModule extends SimpleModuleBase {
      */
     to: string | Date | number;
   }): Date {
+    if (!options) {
+      throw new FakerError('Both from and to must be valid dates.');
+    }
+
     const { from, to } = options;
+    const fromMs = new Date(from).getTime();
+    const toMs = new Date(to).getTime();
+
+    if (Number.isNaN(fromMs) || Number.isNaN(toMs)) {
+      throw new FakerError('Both from and to must be valid dates.');
```

Comment:
Maybe quote the variables?
```suggestion
      throw new FakerError('Both `from` and `to` must be valid dates.');
```